src/App.tsx

```diff
@@ -24,15 +22,22 @@ const App = () => {
   };
 
   return (
-    <main className="t-center">
-      <img src={wrs} className="logo" alt="CRS-logo" />
-      <h1 className="t-header">
-        <span>Click </span>
-        <Button label="Fetch" variant="success" onClick={loadUsers} data-testid="succss-btn" />
-        <span> & let Saga do the rest. </span>
-        <span>Explore <a href="https://sprakash57.github.io/create-react-saga" rel="noopener noreferrer" target="_blank">Docs</a> for more.</span>
-      </h1>
-      {isLoading ? <Alert message="Loading..." /> : renderUser()}
+    <main className='main'>
+      <nav className='main__navbar'>
+        <div className='main__brand'>
+          <h1>Create React Saga</h1>
+          <sub>v{appInfo.version}</sub>
+        </div>
+        <div>
+          <a className='links card' href="https://github.com/sprakash57/create-react-saga" rel="noopener noreferrer" target="_blank">GitHub</a>
+          <a className='links card' href="https://sprakash57.github.io/create-react-saga" rel="noopener noreferrer" target="_blank">Docs</a>
+        </div>
+      </nav>
+      <img src={wrs} className="main__logo" alt="create-react-saga" />
+      <section>
+        <Button label="Click to run Saga" variant="primary" onClick={() => dispatch(fetchUsers())} data-testid="succss-btn" />
```

Comment:
Maybe `variant="success"` would be more fitting here.

Reply:
done